build/build.rs

```diff
@@ -40,3 +57,55 @@ fn main() {
         println!("cargo:rustc-link-lib=c++_shared");
     }
 }
+
+fn build_windows_i686() {
+    // Prepare directories
+    let cargo_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let source_dir = cargo_dir.join("glslang");
+    let install_dir = source_dir.join("install");
```

Comment:
This must be inside `OUT_DIR`. You are not allowed to change the source directory, otherwise cargo won't let you publish.